css/backoffice/components/_field.scss

```
@@ -253,3 +253,6 @@ $ibo-field--enable-bulk--checkbox--margin-left: $ibo-spacing-300 !default;
   margin-left: $ibo-field--enable-bulk--checkbox--margin-left;
 }
 
+.ibo-input-select--action-buttons a  {
+  color: $ibo-field--value--color;
```

Can you try this? This helper aims at forcing an anchor to reuse its container color instead of the "default" anchor color. (Don't apply the suggestion without testing, I'm not sure 😁)

```suggestion
  @extend %ibo-hyperlink-inherited-colors;
```